unittest/log.h

```diff
@@ -29,14 +29,14 @@ static inline std::ostream& LOG(enum LogLevel level)
   switch (level) {
 #if 0
     case DEBUG:
-      std::cout << "[DEBUG] ";
+      std::cout << "\n[DEBUG] ";
       break;
 #endif
     case INFO:
-      std::cout << "[INFO]  ";
+      std::cout << "\n[INFO]  ";
       break;
     case ERROR:
-      std::cout << "[ERROR] ";
+      std::cout << "\n[ERROR] ";
```

Comment:
@Shreeshrii, did you find the implementation which is used by Google, and does that implementation add line feeds like that?

Reply:
Google might be using the implementation in glog - https://github.com/google/glog/blob/master/src/windows/glog/logging.h

I added the linefeed because I thought it might increase readability. It could probably be replaced by a space.

Reply:
I see. The Google implementation adds a linefeed at the end if the log string does not already end with one. As Tesseract only has a few users of `LOG`, I think the linefeed characters can be added locally when calling `LOG` if needed. I suggest to remove the 3rd commit, at least for now.

Reply:
@stweil I don't know how to remove the commit. I have added another commit reverting the change.